decompiler/extractor/main.cpp

```diff
@@ -407,6 +432,25 @@ int main(int argc, char** argv) {
       }
       path_to_iso_files = data_dir_path;
     }
+
+    // write out a json file with some metadata for the game
+    std::string buildinfo_json = "{\n";
+    buildinfo_json += "  \"flags\": [";
+    bool first = true;
+    for (const auto& [n, f] : sGameIsoFlagNames) {
+      if (flags & f) {
+        if (first) {
+          first = false;
+        } else {
+          buildinfo_json += ", ";
+        }
+        buildinfo_json += fmt::format("\"{}\"", n);
+      }
+    }
+    buildinfo_json += "]\n";
+    buildinfo_json += "}\n";
+    // something tells me a ps2 game is unlikely to have a json file in root
+    file_util::write_text_file((path_to_iso_files / "buildinfo.json").string(), buildinfo_json);
```

Comment:
Is the use for this JSON file to let you run the extractor without unpacking the ISO, and figure out which game version it is?

Reply:
yeah